public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -247,8 +250,20 @@ export class PanelModel {
     });
   }
 
+  private applyPluginOptionDefaults(plugin: PanelPlugin) {
+    this.options = _.defaultsDeep({}, this.options, plugin.defaults);
+  }
+
   pluginLoaded(plugin: PanelPlugin) {
     this.plugin = plugin;
+    const hasPersistedOptions = !!this.options;
+    const defaultOptions = plugin.defaults;
+
+    if (hasPersistedOptions) {
+      this.applyPluginOptionDefaults(this.plugin);
+    } else {
+      this.options = _.defaultsDeep({}, defaultOptions);
+    }
```

Comment:
not sure I understand this if statement, cant it be replaced with 
this.options =  _.defaultsDeep({}, this.options || {}, plugin.defaults);

And I think this needs a if (!this.plugin.angularPanelCtrl) or some check so that it only happens on react panels